### PR TITLE
New fields added

### PR DESCRIPTION
**Pull request by miquelbotanch**
All new fields are optional

Group Header:
NifSuffix Company Id to identify Company (Mandatory at Spain)
Payment: 
CreditorAddress
CreditorAddress (2nd line)
CreditorCountry
IsSepa (for sepa and non sepa payments, they can be mixed in the xml)
CtgyPurp to identify if the payment is a salary or supplier payment (all available options are in comments) the good part is that using CtgyPurp at payment level allows you to put any kind of payment together in the same XML, if it's done on the header, you've to create an XML per purpose.

### CODE REVIEW

File: docs/install.rst [hunk 1]
```diff
@@ -16,7 +16,7 @@ To install SEPA PHP, simply require it using Composer:
 
 .. code-block:: bash
 
-    composer require perryfaro/sepa
+    composer require aculvi/sepa
```
Comment (perryfaro) [inline docs/install.rst:19]: Can you change this back to perryfaro instead of aculvi?

File: composer.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-    "name": "perryfaro/sepa",
+    "name": "aculvi/sepa",
     "type": "library",
     "description": "Creates an XML file for a Single Euro Payments Area (SEPA) Credit Transfer.",
     "keywords": ["banking", "sepa", "xml", "SCT"],
     "license": "MIT",
     "authors": [
         {
-            "name": "Perry Faro"
+            "name": "Alan Culvi"
```
Comment (perryfaro) [inline composer.json:9]: Can you change this back to perryfaro instead of aculvi?

File: composer.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
 {
-    "name": "perryfaro/sepa",
+    "name": "aculvi/sepa",
```
Comment (perryfaro) [inline composer.json:2]: Can you change this back to perryfaro instead of aculvi?

**Review by perryfaro — CHANGES_REQUESTED** [review]

**Comment by miquelbotanch** [comment]
Sorry but, could you tell me why my PR has not been accepted  ?
I would like to continue using your repository, but unfortunately in Spain we're forced to use that fields, so I need to add it. in case that in other countries is not necessary, it simply doesn't affect the result XML.

Another stupid limitation here is that you cannot mix SALA and SUPP in the same XML (is pretty simple to solve and create two XML instead of one) but is curious how the simply ignore the standard...

**Pull request closed without merging**